lib/dialects/oracledb/index.js

```diff
@@ -165,6 +180,58 @@ class Client_Oracledb extends Client_Oracle {
     return connection.release();
   }
 
+  // In Oracle, we need to check the version to dynamically determine
+  // certain limits.
+  checkVersion(connection) {
```

Comment:
why not make this method async?

Reply:
Probably makes sense to mark it async, even though I do still need to wrap the call to execute in a promise due to node-oracledb's use of a callback-style of interface. Makes that top portion a bit more readable and it makes it more obvious that the whole function has an async behavior. I'll add that change.